templates/component/stories.tsx.hbs

```diff
@@ -5,5 +5,5 @@ import { {{pascalCase name}} } from '.';
 export default { title: 'Components|{{pascalCase name}}' };
 
 export const Standard = () => (
-	<{{pascalCase name}}/>
+<{{pascalCase name}}/>
```

Comment:
Just reset this file, we need the space ;)

Reply:
Done